centreon/tests/e2e/features/Dashboards/03-dashboard-properties-edition/index.ts

```diff
@@ -45,11 +45,7 @@ Given(
 );
 
 When('the user selects the properties of the dashboard', () => {
-  const dashboardToEdit = dashboardsOnePage[dashboardsOnePage.length - 2];
-  cy.contains(dashboardToEdit.name)
-    .parent()
-    .find('button[aria-label="edit"]')
-    .click();
+  cy.get('button[aria-label="Edit properties"]').eq(3).click();
```

Comment:
```suggestion
      cy.getByLabel({ label: 'Edit properties', tag: 'button' }).eq(3).click();
```
should be applied to every similar change for consistency